core/src/main/java/co/aikar/commands/ACFPatterns.java

```diff
@@ -31,7 +31,7 @@
 import java.util.regex.Pattern;
 
 @SuppressWarnings("WeakerAccess")
-final class ACFPatterns {
+public final class ACFPatterns {
```

Comment:
There's no reason to expose this. I don't want to consider this public API. It's trivial for someone to maintain their own pattern class like this.